.github/workflows/ppc64le.yml

```diff
@@ -0,0 +1,36 @@
+name: ppc64le
+on:
+  push:
+    branches:
+      - master
+      - bytestring-0.11
+  pull_request: {}
```

Comment:
I assume this turns off building PRs? Why? 12 minutes doesn't seem too bad to me.

Reply:
The syntax is confusing indeed. It means that all PRs are validated.

Reply:
```suggestion
  pull_request: {} # Validate all PRs
```